backend/corpora/sync_env_with_prod/app.py

```diff
@@ -0,0 +1,43 @@
+import json
+import boto3
+import subprocess
+
+
+def copy_relational_db(event, context):
+    print('Starting copy')
+    db_uri = generate_db_uri()
+    copy_rds_data(db_uri)
+    # todo update to write to prod after confirming code works in dev
+    write_to_s3('cellxgene-db-dump-dev')
+    return {
```

Comment:
Maybe add a try/catch and return an appropriate error? Not super necessary but could help with debugging issues